src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx

```diff
@@ -0,0 +1,207 @@
+* For an example spreadsheet showing how one might plan out your users' roles and account access, see the [Access grant planning spreadsheet](https://docs.google.com/spreadsheets/d/1FnguDXRUX9FGY14oV4Gx6O08v4vNC2Pv0GGCsU7Pxuw/edit?usp=sharing). 
+
+
+## Overview [#overview]
+
+This tutorial will walk you through:
+
+1. [Organization creation](#create-organization)
+2. [How to add accounts](#add-accounts)
+3. [How to set up an authentication domain](#auth-domains)
+4. [How to set up custom roles](#roles)
+5. [How to set up access grants](#group-access) (give groups access to roles and accounts)
+6. [How to add users](#add-users)
+
+## Step 1: Organization creation [#create-organization]
+
+If you're reading this, you likely already have a New Relic organization. When you [sign up for New Relic](https://newrelic.com/signup), your New Relic organization is created. The organization structure represents a New Relic customer: it's what contains everything relevant to a customer's use of New Relic: their accounts, their users, and their data.
+
+When a New Relic organization is created, it contains a single account. A Standard edition organization can only have a single account, but Pro and Enterprise edition organizations can add more accounts. An account can be considered a workspace. For example, you might have an account for a specific app, or a set of related hosts and services for a specific initiative or project. Each account has its own [account ID](/docs/accounts/accounts-billing/account-structure/account-id), and that ID is used for some account-specific tasks, like making API calls.
+
+When your organization is created, it has several default "access grants," which grant the two available default groups access to specific roles, and a specific scope of accounts. When you add users via the UI, there are two default groups that you can assign your users to:
+
+* **Admin**: can use and configure observability features for that initial account, and the ability to view and configure organization-level settings (like adding accounts or managing users).  
+* **User**: can use and configure observability features, without the higher level organization capabilities. 
+
+You can see the default access grants created for these groups by going to the **Organization and access** UI: 
+
+<img
+  title="New Relic organization and access UI - default access grants"
+  alt="New Relic organization and access UI - default access grants"
+  src={orgAccessUI}
+/>
+
+You can see how the **User** group has the **All product admin** role and access to that initially created account. And you can see how the **Admin** group has four access grants, reflecting its greater capabilities. The "Default" next to those group names refers to them being in the original, default authentication domain (we'll talk more about that later). 
+
+Pro and Enterprise edition organizations are able to add custom groups, or bring in groups from their identity provider. 
```

Comment:
Is there a doc we can link out to for this? Seems like a complex topic and a bit weird to leave at just that.